package.json

```diff
@@ -77,7 +77,7 @@
     "eslint-plugin-promise": "^4.2.1",
     "eslint-plugin-standard": "^5.0.0",
     "eslint-plugin-vue": "^7.5.0",
-    "husky": "^3.0.0",
+    "husky": "^3.1.0",
```

Comment:
Is this required for this PR?

Reply:
@yeniatencio please be careful not to add unrelated dependency updates to your PR's

Reply:
Done.